Makefile

```
@@ -0,0 +1,104 @@
+.PHONY: diagrams lint-md fix-md link-check schemas schema-compile schema-validate schema-negative pre-commit
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)